worker.py

```
@@ -0,0 +1,100 @@
+    def __init__(self, SUPABASE_KEY, SUPABASE_URL, videos_bucket='video'):
+        self.upload_folder = os.path.join("/tmp", 'uploads')
+        os.makedirs(self.upload_folder, exist_ok=True)
+
+        executors = {'default': ThreadPoolExecutor(max_workers=1)}
+        self.scheduler = BackgroundScheduler(executors=executors)
+        self.scheduler.start()
+
+        self.SUPABASE_KEY = SUPABASE_KEY
+        self.SUPABASE_URL = SUPABASE_URL
+        if not self.SUPABASE_URL or not self.SUPABASE_KEY:
+            raise ValueError("SUPABASE_URL or SUPERBASE_ANON_KEY is not set. Supabase operations will fail.")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)